transmat/mixins/fshash/bucket_memory.go

```diff
@@ -94,3 +125,9 @@ func (i *memoryBucketIterator) NextChild() treewalk.Node {
 func (i memoryBucketIterator) Record() Record {
 	return i.lines[i.this]
 }
+
+type memoryBucketByFilename MemoryBucket
+
+func (a memoryBucketByFilename) Len() int           { return len(a.records) }
+func (a memoryBucketByFilename) Swap(i, j int)      { a.names[i], a.names[j] = a.names[j], a.names[i] }
+func (a memoryBucketByFilename) Less(i, j int) bool { return a.names[i] < a.names[j] }
```

Comment:
Don't think this should be needed?  Or if it is, then some other sorter somewhere should've disappeared?

Reply:
The other sort is now unused, it will be removed.